package-scripts.js

```diff
@@ -12,13 +12,13 @@ const path = require('path')
 sh.set('-e')
 
 setColors(['dim'])
-
-const testTypes = ['']
+process.env.TEST_SERIES = '1'
+const testTypes = ['', 'other']
```

Comment:
Does `unit` make more sense that `other` here?

Reply:
`other` is really not unit tests - these are the ones that hit aws.

Reply:
`integration`?

Reply:
I can do that